src/handler/template.ts

```diff
@@ -103,12 +136,21 @@ class HandlerTemplate implements TemplateContract {
     const status = parseInt(response?.code!);
     const hasResponseBody = status !== 204 && status < 300; // 성공(2xx) 응답 중 204만 제외
     const responseType = hasResponseBody && response.responses ? Object.keys(response.responses)[0] : undefined;
-    const body = hasResponseBody ? `${identifier ? `await ${identifier}(info)` : 'undefined'}` : 'undefined';
+    const isStreamingResponse = responseType === 'text/event-stream';
+
+    let body: string;
+    if (!hasResponseBody) {
+      body = 'undefined';
+    } else if (isStreamingResponse) {
+      body = `createStreamingResponse(await ${identifier}(info))`;
+    } else {
+      body = identifier ? `await ${identifier}(info)` : 'undefined';
+    }
```

Comment:
```suggestion
		const body = (() => {
			if (!hasResponseBody) return "undefined";
			if (isStreamingResponse)
				return `createStreamingResponse(await ${identifier}(info))`;

			return identifier ? `await ${identifier}(info)` : "undefined";
		})();
```

현재는 약간 가독적이지 않은 느낌이 드는데, 이렇게 바꾸는 것은 어떨까요?? 
~~let을 쓰면 죽는 병에 걸린 사람...~~